modules/environment/aws/iam/cert-manager/main.tf

```diff
@@ -26,34 +26,27 @@ resource "aws_iam_role" "cert_manager_service_account" {
 EOF
 }
 
-#tfsec:ignore:aws-iam-no-policy-wildcards
 resource "aws_iam_role_policy" "cert_manager" {
   name = "${var.cluster}-cert-manager"
   role = aws_iam_role.cert_manager_service_account.name
 
-  policy = <<EOF
-{
- "Version": "2012-10-17",
- "Statement": [
-    {
-        "Effect": "Allow",
-        "Action": "route53:GetChange",
-        "Resource": "arn:aws:route53:::change/*"
-    },
+  policy = jsonencode(
     {
-        "Effect": "Allow",
-        "Action":  [
-          "route53:ChangeResourceRecordSets",
-          "route53:ListResourceRecordSets"
-        ],
-        "Resource": "arn:aws:route53:::hostedzone/*"
-    },
-    {
-        "Effect": "Allow",
-        "Action": "route53:ListHostedZonesByName",
-        "Resource": "*"
-    }
- ]
-}
-EOF
+      "Version" : "2012-10-17",
+      "Statement" : [
+        {
+          "Effect" : "Allow",
+          "Action" : "route53:GetChange",
+          "Resource" : "${formatlist("arn:aws:route53:::change/%s", var.hosted_zone_ids)}"
```

Comment:
this doesn't look right.  I would expect this to be a "change ID", not a hosted zone ID.  I think this needs to be left as a wildcard